.eslintrc

```
@@ -54,7 +54,7 @@
 					}
 				],
 				"curly": "error",
-				"@delagen/deprecation/deprecation": "warn",
+				"deprecation": "warn",
```

```suggestion
				"deprecation/depreciation": "warn",
```

see https://github.com/gund/eslint-plugin-deprecation#setup